extensions/reactive-mssql-client/deployment/pom.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-reactive-mssql-client-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-reactive-mssql-client-deployment</artifactId>
+
+    <name>Quarkus - Reactive MS SQL Client - Deployment</name>
```

Comment:
Same question about the name.
It looks from their page that the name is just "SQL Server" but SQL Server Client is weird... SQL Client is not exact. I understand how you ended up with MS SQL Client. I would still expand MS.

Reply:
+1 to expand here and in the description for sake o clarity.

In other areas the short hand is fine, as it's commonly referred to as MS SQL by its users (which we are addressing).

Reply:
Done